lib/tello/dsl.rb

```diff
@@ -36,14 +55,14 @@ def in_move_range?(v)
     end
 
     # Takeoff and land
-    [:takeoff, :land].each do |cmd|
+    [:takeoff, :land, :streamon, :streamoff].each do |cmd|
       define_method cmd do
         Tello::Client.return_bool send("#{cmd.to_s}")
       end
     end
 
     # Move in a given direction
-    [:up, :down, :left, :right, :forward, :backward].each do |cmd|
+    [:up, :down, :left, :right, :forward, :back].each do |cmd|
```

Comment:
Oh no, have I had the wrong command for flying backwards all this time? 🤦‍♂️ Good catch.